packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -168,27 +162,21 @@ export const useArbTokenBridge = (
     React.Dispatch<void>
   ]
 
-  const [ERC721Cache, setERC721Cache, clearERC721Cache] = useLocalStorage<
-    string[]
-  >('ERC721Cache', []) as [
-    string[],
-    React.Dispatch<string[]>,
-    React.Dispatch<void>
-  ]
+  const [ERC721Cache, , clearERC721Cache] = useLocalStorage<string[]>(
```

Comment:
Previously, we were calling `setERC721Cache` with no parameters to clear, was that intended? @spsjvc